### PR TITLE
refactor(extensions/button): migrate button extensions to standalone

### PR DESCRIPTION
**Pull request by damingerdai**
relate: #216

### CODE REVIEW

File: projects/extensions/button/button-loading.ts [hunk 1]
```diff
@@ -54,7 +55,7 @@ export class MatButtonLoading implements OnChanges {
     if (!changes.loading) {
       return;
     }
-
+    console.log('ngOnChanges', changes);
```
Comment (nzbin) [inline projects/extensions/button/button-loading.ts:58]: The console should be removed.

File: package.json [hunk 2]
```diff
@@ -11,8 +11,9 @@
     "start": "npm run serve:dev-app",
     "build": "npm run build:dev-app",
     "test": "ng test",
+    "test:lib:ci": "ng test extensions --browsers=ChromeHeadless --watch=false --code-coverage",
     "e2e": "ng e2e",
-    "lint": "npm run lint:ts && npm run lint:scss",
+    "lint": "npm run lint:ts && npm run lint:scss",
```
Comment (nzbin) [inline package.json:16]: Please remove the empty space.

File: package.json [hunk 3]
```diff
@@ -11,8 +11,9 @@
     "start": "npm run serve:dev-app",
     "build": "npm run build:dev-app",
     "test": "ng test",
+    "test:lib:ci": "ng test extensions --browsers=ChromeHeadless --watch=false --code-coverage",
```
Comment (nzbin) [inline package.json:14]: Is it better naming `test:lib`?

**Review by nzbin — COMMENTED** [review]
@damingerdai  Please check the comments.

**Comment by damingerdai** [comment]
@nzbin thanks for your review. please review.

**Pull request merged**